src/knex-processor.ts

```diff
@@ -0,0 +1,54 @@
+import * as Knex from "knex";
```

Comment:
@Exelord A regular `import` (no wildcard) should work:

```suggestion
import Knex from "knex";
```

If it doesn't, can you verify that `allowSyntheticImports` is set to `true` on the `tsconfig.json` file?

Reply:
It is present by the `*` is still required to have it working :/